functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -44,6 +44,12 @@ function Install-DbaMaintenanceSolution {
         Specifies the path to a local file to install Ola's solution from. This *should* be the zip file as distributed by the maintainers.
         If this parameter is not specified, the latest version will be downloaded and installed from https://github.com/olahallengren/sql-server-maintenance-solution
 
+    .PARAMETER NoDataLoss
+        Normally, when the ReplaceExisting flag is used, it will drop the
+        CommandLog table and re-create it. When this flag is also passed, it
+        will instead rename the old table with a time stamp which will allow
+        you to do a migration of existing data to the new table.
+
```

Comment:
If we are going to offer a parameter that can end up making multiple copies of a large table are we going to offer something to clean it up as well?